src/lib/Tests/Menu/ReorderByOrderNumberListenerTest.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Tests\Menu;
+
+use EzSystems\EzPlatformAdminUi\Menu\Event\ConfigureMenuEvent;
+use EzSystems\EzPlatformAdminUi\Menu\Listener\ReorderByOrderNumberListener;
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use Knp\Menu\MenuFactory;
+use PHPUnit\Framework\TestCase;
+
+final class ReorderByOrderNumberListenerTest extends TestCase
+{
+    public function testUnorderedMenuListUntouched(): void
+    {
+        $factory = new MenuFactory();
+        $menu = $this->recursiveCreateMenuChildren(
+            $factory->createItem('menu'),
+            $factory, [
+                ['name' => 'first'],
+                ['name' => 'second'],
+                ['name' => 'third'],
+                ['name' => 'fourth'],
+                ['name' => 'fifth'],
+            ]
+        );
+
+        $eventMock = $this->getConfigureMenuEventMock($menu);
+
+        $listener = new ReorderByOrderNumberListener();
+        $listener->reorderMenuItems($eventMock);
+
+        $expected = [
+            'first',
+            'second',
+            'third',
+            'fourth',
+            'fifth',
+        ];
+
+        $this->assertEquals($this->getChildrenNames($menu), $expected);
```

Comment:
per convention it's the other way around: 
```suggestion
        $this->assertEquals($expected, $this->getChildrenNames($menu));
```
(see the method prototype).

Reply:
@alongosz True, fixed.